records/2022-Q4-activity-report.md

```diff
@@ -0,0 +1,420 @@
+In particular, it is the basis for TASTy-MiMa.
+More information can be found [in the readme of tasty-query](https://github.com/scalacenter/tasty-query#readme).
+
+In this quarter, we made significant progress on tasty-query, and released several in-progress versions, the latest being 0.5.5.
+tasty-query is already used in the Scala Debug Adapter to provide smart step-into filters for Scala 3 code, as well as in the in-progress TASTy-MiMa.
+
+### TASTy-MiMa
+
+For Scala 3.
+
+TASTy-MiMa is a new, in-progress tool that checks TASTy-compatibility across library versions.
+It is similar to MiMa, which works at the binary compatibility level.
+
+The project can be found at https://github.com/scalacenter/tasty-mima.
+It is based on tasty-query.
+
+We implemented most of the core analysis, including:
+
+* Translate types and symbols across two different "contexts" (universes, or classpaths)
+* Missing classes and members
+* Incompatible changes of the kind of members
+* Incompatible changes of the type of term members
+* Incompatible changes in the visibility of members, as well as of openness of classes
+
+The only core changes left to implement are class headers and type members.
+
+After that, we will move on to implementing an sbt and/or Mill plugin to make it available to library maintainers.
+
+## Education and Documentation
+
+### Scala Website
+
+For Scala 2 and Scala 3.
+
+The [scala-lang.org](https://scala-lang.org) homepage is an important destination for both Scala users and non-Scala users, it is the most visited page at 44K visitors per month (the overall site receives 240K per month). Our long term goal is to make the homepage the best destination for both newcomers and returning users to identify the current state of Scala: what it is, what you can do with it, and who is using it.
+
+While we are working on a refreshed front page, we made a usability improvement which was to restore the "Scala versions" to the front page (PR [scala-lang#1435][scala-lang#1435]). This puts at the top of the page the current 3.x and 2.13.x versions and a link to all releases. Previously users would have to go to the "Install" quick link to see this information.
+
+### Scala Documentation Website
+
+For Scala 2 and Scala 3.
+
+The goal of this project is to improve the experience developers have reading the [docs.scala-lang.org website][scala-docs]. We will provide a seamless experience where content users can choose to see content specialised for Scala 2 or Scala 3, or unified when applicable to both. We also want to reduce the number of steps it takes to see content, and offer clear paths to newcomers.
+
+During this quarter, we have continued to migrate to using language version tabs across the site. These tabs show content specialised to either version of the language, typically code, and possibly explanations also. The user's preference is remembered so that all tabs are sychronized across the site to show either Scala 2 or Scala 3 by default.
```

Comment:
```suggestion
During this quarter, we have continued the migration to using language version tabs across the site. These tabs show content specialised to either version of the language, typically code, and possibly explanations also. The user's preference is remembered so that all tabs are sychronized across the site to show either Scala 2 or Scala 3 by default.
```